Dockerfile

```diff
@@ -222,6 +222,54 @@ RUN apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(ls
   && terraform -help \
   && rm -rf /var/lib/apt/lists/*
 
+
+### DART
+
+# Install Dart 
+ENV PUB_CACHE=/opt/dart/pub-cache \
+  PUB_ENVIRONMENT="dependabot" \
+  PATH="${PATH}:/opt/dart/dart-sdk/bin"
+ARG DART_VERSION=2.15.1
+RUN curl --connect-timeout 15 --retry 5 "https://storage.googleapis.com/dart-archive/channels/stable/release/${DART_VERSION}/sdk/dartsdk-linux-x64-release.zip" > "/tmp/dart-sdk.zip" \
+  && mkdir -p "$PUB_CACHE" \
+  && chown dependabot:dependabot "$PUB_CACHE" \
+  && unzip "/tmp/dart-sdk.zip" -d "/opt/dart" > /dev/null \
+  && chmod -R o+rx "/opt/dart/dart-sdk" \
+  && rm "/tmp/dart-sdk.zip" \
+  && dart --version
+# HACK: We use a patched version of "dart pub" that is not in the Dart SDK yet.
+#       Hence, we clone from git and globally install it from local folder.
+#       Now it can be used with: 'dart pub global run pub ...'
+RUN git clone https://github.com/dart-lang/pub.git /opt/dart/pub \
+  && git -C /opt/dart/pub checkout 941191f7f83ad60259348860197cfcdd83bb8e6f \
+  && dart pub global activate --source path /opt/dart/pub \
+  && chmod -R o+r "/opt/dart/pub" \
+  && chown -R dependabot:dependabot "$PUB_CACHE" \
+  && chown -R dependabot:dependabot /opt/dart/pub
```

Comment:
Now that the dependency services API has landed, can we clean this up, or do we still need the manual checkout?

Reply:
Cool, thanks for clarifying!